nextflow_schema.json

```diff
@@ -13,19 +13,35 @@
             "properties": {
                 "input": {
                     "type": "string",
-                    "default": null,
+                    "default": "None",
```

Comment:
Null is probably better as it'd be easier to catch for presence or not later in the pipeline

Reply:
addressed